db/seeds/a_products.js

```diff
@@ -1,22 +1,25 @@
 var faker = require('faker');
 
-exports.seed = function(knex) {
+exports.seed = (knex) => {
   // Deletes ALL existing entries
   return knex('products').del()
     .then(function () {
       return knex.raw('ALTER TABLE ' + 'products' + ' AUTO_INCREMENT = 1')
     })
-    .then(function () {
+    .then( async () => {
       //Create array of fake products
       var fakeProducts = [];
-      for (var i = 0; i < 100; i++) {
+      for (var i = 0; i < 500001; i++) {
         if (i === 0) {
           fakeProducts.push({name: 'Star Wars Super Deluxe 24" Talking Plush: Chewbacca'});
         } else {
           fakeProducts.push({name: faker.random.words(3)});
         }
+        if ( i % 5000 === 0) {
+          await knex('products').insert(fakeProducts);
+          fakeProducts = [];
+        }
```

Comment:
Looks like a good way to batch insert on 5000 increments. Looks like you're using PostgreSQL, but if things end up slowing down too much when you ramp up to 10m records I'd recommend looking into the COPY command, it is PostgreSQL's prefered way to bulk insert.